crates/metrics/src/eigenmetrics.rs

```diff
@@ -13,17 +14,41 @@ pub struct EigenMetrics {
     performance_score: Gauge,
 }
 
-impl EigenMetrics {
-    pub fn new() -> Self {
-        let gauge = Self {
-            performance_score: metrics::register_gauge!("eigen_performance_score"),
+// TODO(supernova) : feeearnedtotal is not turned on yet,so not implemented yet
+// https://github.com/Layr-Labs/eigensdk-go/blob/67787e959b727b115628a34e796df3a9ef42f646/metrics/eigenmetrics.go#L23
+
+#[derive(Debug)]
+pub struct EigenMetricsMetrics {
```

Comment:
does it really need to be called *MetricsMetrics? if yes can you leave an explanation as to why?

Reply:
added an explanation , alternatively we can name this struct as  EigenPerformance also , then the wrapper will be EigenPerformanceMetrics
```
pub struct EigenMetrics {
    /// performance score
    #[metric(
        rename = "eigen_performance_score",
        describe = " A gauge with performance score "
    )]
    performance_score: Gauge,
}
```

Reply:
let's do it then